README.md

```diff
@@ -68,10 +95,79 @@ ensure all the proposed actions are correct before proceeding.
 In order to actually have the utility perform all necessary API calls, the
 `--not-a-drill` flag needs to be supplied.
 
-
-#### Notable parameters:
-  * `--not-a-drill`: if set, the utility will go through the process of
-    executing all the necessary actions (without it, the utility only prints
-    the actions it would take)
+**Notable params:**
   * `--dont-recreate-tenants`: if set, will *not* create destination tenants
-    (they will need to be pre-created by the migration administrator)
+    (they will need to be pre-created by the migration administrator, or with the `migrate tenant` command)
+
+### Assess migration
+The command may be used with any number of migration ids, its purpose is to give to the user
+information about a running or completed migration such as:
+* instance information
+* migration status
+* migration time
+
+```bash
+coriolis-openstack-util --config-file ./path/to/conf.ini assess migration MIGRATION-ID1 MIGRATION-ID2 ...
+```
+**Notable params:**
+  * `--format`: the output format of the migration information: yaml, json(default) or excel
+  * `--excel-filepath`: only if *excel* format specified, file path where the excel will be written.
+
+### Assess instance
+The command may be used with any number of instance names, it fetches relevant instance information for the eventual
+migration.
+
+```bash
+coriolis-openstack-util --config-file ./path/to/conf.ini assess instance INSTANCE_NAME1 INSTANCE_NAME2 ...
+```
+
+**Notable params:**
+  * `--format`: the output format of the migration information: yaml, json(default) or excel
+  * `--excel-filepath`: only if *excel* format specified, file path where the excel file will be written
+
+### Migrate network
+Migrate a network, alongside all of its subnets.
+```bash
+coriolis-openstack-util --config-file .path/to/conf.ini migrate network \
+--src-network-name $NET_NAME \
+--dest-tenant-name $TENANT_NAME \
+--not-a-drill
+```
+### Migrate router
+Migrate a router, with routes to migrated networks according to the `new_network_name_format` config option.
+```bash
+coriolis-openstack-util --config-file ./path/to/conf.ini migrate router \
+--src-router-name $ROUTER_NAME \
+--dest-tenant-name $TENANT_NAME \
+--not-a-drill
+```
+### Migrate security group
+Migrate security group with all of its rules.
+```bash
+coriolis-openstack-util --config-file ./path/to/conf.ini migrate secgroup \
+--src-tenant-name $TENANT_NAME \
+--dest-tenant-name $TENANT_NAME \
+--not-a-drill \
+$SECURITY_GROUP_NAME
+```
+### Migrate subnet
+Migrate subnet from source network to destination migrated network with new name according to the
+`new_network_name` config option.
+```bash
+coriolis-openstack-util --config-file ./path/to/conf.ini migrate subnet \
+--src-network-id $SRC_NETWORK_ID \
+--not-a-drill \
+$SUBNET_NAME
+```
+### Migrate user
+Create new user on destination with name according to the `new_user_name_format` and `new_user_password` config options.
```

Comment:
"Create a new user on the destination OpenStack named according to the `new_user_name_format `. The password the user will be created with can be configured with the `new_user_password` configuration option"